projects/plugins/crm/includes/ZeroBSCRM.Database.php

```diff
@@ -879,6 +880,33 @@ function zeroBSCRM_createTables(){
   COLLATE = ".$collation.";"; 
   zeroBSCRM_db_runDelta($sql);
 
+	// Add table to store automation workflows.
+	// phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+	$sql = 'CREATE TABLE IF NOT EXISTS ' . $ZBSCRM_t['automation-workflows'] . '(
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`zbs_site` INT NOT NULL,
+	`zbs_owner` INT NOT NULL,
+	`name` VARCHAR(255) NOT NULL,
+	`description` VARCHAR(600) NULL DEFAULT NULL,
+	`category` VARCHAR(255) NOT NULL,
+	`triggers` JSON NOT NULL,
+	`initial_step` VARCHAR(255) NOT NULL,
+	`steps` JSON NOT NULL,
+	`active` TINYINT(1) NOT NULL DEFAULT 0,
+	`version` INT(14) NOT NULL DEFAULT 1,
+	`created_at` TIMESTAMP NOT NULL DEFAULT 0,
```

Comment:
MariaDB doesn't seem to have the same CURRENT_TIMESTAMP() limitation, so nice catch!
I do, however, feel like we should never allow" `created_at` to be null, so perhaps using it here makes sense?

Reply:
I preserved the `NOT NULL` in both, `created_at` and `updated_at. Both will be set programmatically on insert and `updated_at` on update. In PHP the int declaration, the default value is `0`, so... should we allow nullable values on both? (See Thomas comment)

Reply:
I strongly prefer not using 0 as a default for timestamps, as that's an actual timestamp. Why not do `NOW()` for `created_at` and `NULL` for `updated_at`?

Also, I'll note that the other tables use `created` and `lastupdated` if we want to be consistent.

Reply:
Okay let's do this:

`created_at`  => `NOT NULL DEFAULT NOW()`
`updated_at` => `DEFAULT NULL`